debian/revpi-repo.install

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/dh-exec
```

Comment:
don't use dh-exec!

Reply:
Right. Needs https://github.com/RevolutionPi/revpi-repo/pull/8 before I can change it here.